aspnetcore/fundamentals/openapi/openapi-comments.md

```diff
@@ -0,0 +1,307 @@
+---
+title: OpenAPI XML documentation comment support in ASP.NET Core
+author: captainsafia
+description: Learn how to integrate XML documentation comments on types by OpenAPI document generation in ASP.NET Core.
+ms.author: safia
+monikerRange: '>= aspnetcore-10.0'
+ms.custom: mvc
+ms.date: 10/26/2024
+uid: fundamentals/openapi/aspnet-openapi-xml
+---
+<!-- backup author: rick-anderson -->
+# OpenAPI XML documentation comment support in ASP.NET Core
+
+OpenAPI XML documentation integration is implemented as a source generator. The source generator runs at compile time and injects code that translates XML comments into OpenAPI metadata. No configuration is required in app code to enable XML documentation comment support. The source generator automatically detects XML documentation comments in the application assembly and any referenced assemblies that have XML documentation enabled.
+
```

Comment:
This might be a good place.
```suggestion

ASP.NET Core processes [XML documentation tags](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags) like: `<c>`, `<code>`, `<list>`, `<para>`, `<paramref>`, `<typeparamref>`, `<see>`, and `<seealso>`.
For XML documentation tags that use references to other elements, like `<see cref="SomeOtherType">`, the implementation strips out the XML tag and maps the reference to plain text for inclusion in the OpenAPI document.

```